CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [0.18.13](https://github.com/brave/browser-laptop/releases/tag/v0.18.13dev)
+ - Added a welcome screen for first time users. ([#9423](https://github.com/brave/browser-laptop/issues/9423))
+ - Added Chromium spell checker. ([#9880](https://github.com/brave/browser-laptop/issues/9880))
+ - Added an option to disable "Top Sites" in autocomplete. ([#4977](https://github.com/brave/browser-laptop/issues/4977))
+ - Added "Hide Lower" button on Brave Payments. ([#9137](https://github.com/brave/browser-laptop/issues/9137))
+ - Optimized sending messages only to needed windows instead of all windows. ([#9701](https://github.com/brave/browser-laptop/issues/9701))
+ - Fixed showing the URL in the location bar for invalid URLs (instead of about:error). ([#2025](https://github.com/brave/browser-laptop/issues/2025)) 
+ - Fixed opening a context menu freezes video (ex: Youtube). ([#5470](https://github.com/brave/browser-laptop/issues/5470))
```

Comment:
@bsclifton @bbondy in which PR did we fix this. I think that async menu was already implemented in Jun,probably in 0.16.x